learn/concepts/federation.md

```diff
@@ -36,7 +37,7 @@ Please note that your federation server **must** use `https` protocol.
 
 The federation URL specified in your stellar.toml file should accept an HTTP GET request and issue responses of the form detailed below.
 
-Instead of building your own server you can use [`federation`](https://github.com/stellar/federation) server built by Stellar.
+Instead of building your own server you can use the [`federation server`](https://github.com/stellar/federation) built by StellarOrg. This software is still in alpha and is not ready for production.
```

Comment:
the federation server is ready for people to use. I would say SDF or stellar.org rather than StellarOrg